README.old.md

```diff
@@ -0,0 +1,75 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
```

Comment:
Kindly use just one `README.md` file, try to use this one or remove the other one.